subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3997,6 +3999,14 @@ int lwm2m_engine_context_close(struct lwm2m_ctx *client_ctx)
 		}
 	}
 
+	for (i = 0, msg = messages; i < CONFIG_LWM2M_ENGINE_MAX_MESSAGES;
+	     i++, msg++) {
+		memset(msg, 0, sizeof(struct lwm2m_message));
+	}
+
+	coap_pendings_clear(client_ctx->pendings, CONFIG_LWM2M_ENGINE_MAX_PENDING);
```

Comment:
@retfie Unfortunately, moving the CONFIG_LWM2M_ENGINE_MAX_PENDING up to the line above has broken the line length rule for Zephyr :/

I was imagining something like this:
```
	coap_pendings_clear(client_ctx->pendings,
			    CONFIG_LWM2M_ENGINE_MAX_PENDING);
```

Reply:
@mike-scott  sorry, short tab to blame :/ . Fixed.